### PR TITLE
Author build process

### PR DESCRIPTION
**Pull request by tgandrews**
This change includes:
1. Allow author to control folders before docker build run.
2. Remove author build file as we are using cypress and they provide up
to date version with node 10.

Dependencies:
- [x] -  https://github.com/ONSdigital/eq-author-app/pull/43

### CODE REVIEW

File: build_containers/author-build.dockerfile [hunk 1]
```diff
@@ -1,13 +0,0 @@
-FROM ubuntu:16.04
```
Comment (jonnyshaw89) [inline build_containers/author-build.dockerfile:1]: This is used in other builds so we shouldn't remove this 
https://github.com/ONSdigital/eq-deploy/pull/45/files#diff-c5a2c990f03bcdb3e6d4e81cf49d64f5R142

File: eq.yml [hunk 2]
```diff
@@ -204,28 +205,32 @@ jobs:
...
         - |
-          cd eq-author-app
-          eq_author_tag=$(cat ../eq-author-app/.git/HEAD | xargs echo -n)
-          echo "{\"APPLICATION_VERSION\":\""$eq_author_tag"\"}" > ../build_args/args
+          pushd eq-author-app
+          EQ_AUTHOR_VERSION=$(cat .git/HEAD | xargs echo -n)
+          echo "{\"APPLICATION_VERSION\":\""$EQ_AUTHOR_VERSION"\"}" > ../build_args/args
+          ./councourse.sh
+          popd
+          cp -R eq-author-app/* built-eq-author-app
```
Comment (jonnyshaw89) [inline eq.yml:214]: add `built-eq-author-app` as an output

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: build_containers/author-build.dockerfile (hunk 1 above)
Comment (tgandrews) [reply build_containers/author-build.dockerfile]: The only places I could see it used was in the smoke tests and the author build. Is there somewhere else?

Comment (tgandrews) [reply build_containers/author-build.dockerfile:1]: Removed from all locations.

File: deploy.yml [hunk 3]
```diff
@@ -155,8 +156,13 @@ jobs:
           mkdir eq-author-app
           tar -xzf source.tar.gz -C eq-author-app --strip-components=1
 
-          eq_author_tag=$(cat tag)
-          echo "{\"APPLICATION_VERSION\":\""$eq_author_tag"\"}" > ../build_args/args
+          EQ_AUTHOR_VERSION=$(cat tag)
+          echo "{\"APPLICATION_VERSION\":\""$EQ_AUTHOR_VERSION"\"}" > ../build_args/args
+
+          pushd eq-author-app
+          ./concourse.sh
```
Comment (jonnyshaw89) [inline deploy.yml:163]: typo here and we should use `path: bash`

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]
We don't need to supply args to the author image build
https://github.com/ONSdigital/eq-deploy/pull/45/commits/d7e772e4fa1b69cdb79ba73311dc225d496e9911#diff-d25a786b1f55fcf30f80ba1add045324R220

**Review by jonnyshaw89 — APPROVED** [review]

**Comment by jonnyshaw89** [comment]
Squash commits?

**Review by jonnyshaw89 — APPROVED** [review]

**Review by SamGodwin2 — APPROVED** [review]

**Pull request merged**